_board/uchip.md

```diff
@@ -47,6 +47,13 @@ You can program uChip using virtually any IDE, in many languages (CircuitPython
 * Standards narrow-DIP footprint: 0.3” (7.62 mm) row spacing, 0.1” (2.54) pin spacing.
 * Pinout standard logic CMOS compatible: power and GND are on pin 16 and 8, so you can also emulate some 16 pin CMOS ICs (4000 and 74HC series)!  
 
+**Additional Notes:**
+
+* The uChip does not have an RGB LED.
+* The on-board LED is amber, rather than green.
```

Comment:
I don't get the relevance here. For most use cases the color of the status LED doesn't matter.

Reply:
see previous comment

Reply:
The color was mentioned in case it _does_ matter.

Reply:
Mentioning this is fine with us.